miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/file/MiniAppFileChooser.kt

```diff
@@ -49,6 +50,17 @@ class MiniAppFileChooserDefault(var requestCode: Int) : MiniAppFileChooser {
             if (fileChooserParams?.mode == WebChromeClient.FileChooserParams.MODE_OPEN_MULTIPLE) {
                 intent?.putExtra(Intent.EXTRA_ALLOW_MULTIPLE, true)
             }
+            // Uses Intent.EXTRA_MIME_TYPES to pass multiple mime types.
+            fileChooserParams?.acceptTypes?.let { acceptTypes ->
+                if (!acceptTypes.equals("") && acceptTypes.size > 1) {
```

Comment:
`acceptTypes.size > 1` means that it won't work if only one type is passed, i.e. `accept=".jpg"`. Should it be like this instead?
```suggestion
                if (!acceptTypes.equals("") && acceptTypes.isNotEmpty()) {
```